source/_templates/installations/manager/configure_indexer_connection.rst

```diff
@@ -1,12 +1,14 @@
 .. Copyright (C) 2015, Wazuh, Inc.
 
+By default, the host is set to localhost: ``<host>https://0.0.0.0:9200</host>``. Replace ``0.0.0.0`` with your Wazuh indexer node IP address or hostname.
+
 .. code-block:: xml
```

Comment:
Check alignment. We use three spaces indentation and we should check the configuration block sample alignment as well. Maybe something like this

```
.. code-block:: xml
   :emphasize-lines: 4

   <indexer>
     <enabled>yes</enabled>
     <hosts>
       <host>https://0.0.0.0:9200</host>
     </hosts>
     <ssl>
       <certificate_authorities>
         <ca>/etc/filebeat/certs/root-ca.pem</ca>
       </certificate_authorities>
       <certificate>/etc/filebeat/certs/filebeat.pem</certificate>
       <key>/etc/filebeat/certs/filebeat-key.pem</key>
     </ssl>
   </indexer>

```

Reply:
Done!

Reply:
Sorry, I wasn't clear. I mean the alignment of Sphynx code only, not code inside code blocks. For example, we use

```
.. code-block:: xml
   :emphasize-lines: 4

   some code here
```

rather than, say

```
.. code-block:: xml
     :emphasize-lines: 4

 some code here

```